modules/morphology/src/test/resources/data/neurosciencegraph/morphology/reconstructedwholebraincell/all-fields.json

```diff
@@ -0,0 +1,34 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/data/v0.1.0"
+  ],
+  "@type": [
+    "prov:Entity",
+    "nsg:ReconstructedWholeBrainCell"
+  ],
+  "name": "04392-04406-X11579-Y24237",
+  "providerId": "04392-04406-X11579-Y24237",
+  "brainLocation": {
+    "brainRegion": {
+      "@id": "http://voxels.nexus.apps.bbp.epfl.ch/api/analytics/atlas/releases/1192CEAC-17C5-4A74-B88C-48065C048678/filters/brain_region/262",
+      "label": "Reticular nucleus of the thalamus"
+    },
+    "atlasSpatialReferenceSystem" : {
+      "@id": "{{base}}/data/neurosciencegraph/atlas/atlasspatialreferencesystem/v0.1.0/d12f3442-8f5b-4df6-aef0-485a69871b8b",
```

Comment:
Let use the official Allen term for this brain parcel:

http://api.brain-map.org/api/v2/data/Structure/262